tasks/build/docker/packages/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG node_version
+FROM node:$node_version
```

Comment:
I believe there are alpine images available for this, just append `-alpine`.

Reply:
can't yet.  the node image is based on 3.4 and rpm isn't available https://pkgs.alpinelinux.org/packages?name=rpm&branch=&repo=&arch=&maintainer=